# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.h
# Commit: 0531e0fd8b47
# Full Hash: 0531e0fd8b47b52fe15bc3f2c6a337b37a4c93fc
# Author: stransky <stransky@redhat.com>
# Date: 2022-07-26 21:44:38
# Regressor Bug: 1777664
# File Overlap Count: 1
# Description:
#   Bug 1777664 [Wayland] Don't pause compositor when nsWindow is hidden r=emilio
#   
#   We should not pause compositor as compositing is driven by refresh driver and compositor pause leads to browser freeze.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152689
# ==============================================================================

diff -r 8a1503365add -r 0531e0fd8b47 widget/gtk/nsWindow.h
--- a/widget/gtk/nsWindow.h	Tue Jul 26 14:56:47 2022 +0000
+++ b/widget/gtk/nsWindow.h	Tue Jul 26 14:56:47 2022 +0000
@@ -486,7 +486,6 @@
   void EnableRenderingToWindow();
   void DisableRenderingToWindow();
   void ResumeCompositorHiddenWindow();
-  void PauseCompositorHiddenWindow();
   void WaylandStartVsync();
   void WaylandStopVsync();
   void DestroyChildWindows();
@@ -790,7 +789,6 @@
   bool IsWidgetOverflowWindow();
   void RemovePopupFromHierarchyList();
   void ShowWaylandWindow();
-  void HideWaylandWindow();
   void HideWaylandPopupWindow(bool aTemporaryHidden, bool aRemoveFromPopupList);
   void HideWaylandToplevelWindow();
   void WaylandPopupHideTooltips();
